custom_reports/http_api/index.yaml

```diff
@@ -24,8 +24,17 @@ paths:
     $ref: ./paths/sessions.yaml
   /api/analytics/v1/events/:
     $ref: ./paths/events.yaml
+  /api/analytics/v1/goals/:
+    $ref: ./paths/goals.yaml
+  /api/analytics/v1/goals/{id}/:
+    $ref: ./paths/goals_detail.yaml
```

Comment:
I'd say object management APIs should have separate section in our docs from the reporting APIs

Reply:
Sure.